### DISCUSSION TITLE
Error in detecting people (Tapo C320WS)

### DISCUSSION THREAD

**Discussion by AiramsRT** [JurajNyiri/HomeAssistant-Tapo-Control#727] [General]
Hello.

I just added this camera to the integration, the motion, animal, etc. modes work for me, but the person detection mode doesn't work for me, do you know how to fix it? I put it in high, low, medium mode... it doesn't matter, none of them detect it and in the official tapo app I see the activation of the option.

The thing is that from the tapo app it doesn't detect it either, I don't know if it's because of a firmware error when resetting the camera, but if I give it access to the internet it works but it stops working with the integration, any ideas?

Sorry for my English, it's with google translate

**Comment by JurajNyiri** [comment]
Hello, if the setting is set and does not work for detections you will need to contact TPLink as there is nothing to fix on integration side.

**Comment by AiramsRT** [comment]
If I disable the integration and leave it only with the tapo application, the detection works perfectly, that's the strange thing

**Comment by AiramsRT** [comment]
To test, I have set a light and sound alarm, only activating the person detection. If a person is detected, the alarm sounds, but the integration does not show the person detection in the log, however, the camera detects well because the alarm goes off.

**Reply by JurajNyiri** [reply]
Integration currently does not support detection events for AI events. Work is ongoing at https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/304

**Reply by AiramsRT** [reply]
Ok, so that's my problem, the only thing I need is motion detection because if I select motion it creates a lot of false positives due to vehicle lights. I'll be on the lookout for future updates. Thanks
